src/mip/coefficient_strengthening.jl

```diff
@@ -0,0 +1,99 @@
+function maximal_activity(row::RowOrCol{T}, j::Int, ucol::Vector{T}, lcol::Vector{T})::T where {T}
+    # compute the sup of row i (execpt the integer variable j)
+
+    # only compute the maximal activity of any variables except j
+    nonzero_index = [i for i in 1:length(row.nzind) if row.nzind[i] != j]
+    nonzero_value = row.nzval[nonzero_index]
+
+    pos_coef = deepcopy(nonzero_value)
+    neg_coef = deepcopy(nonzero_value)
+
+    for i in 1:length(nonzero_value)
+        if nonzero_value[i] < 0
+            pos_coef[i] = 0
+        else
+            neg_coef[i] = 0
+        end
+    end
+    sup = ucol[nonzero_index]'pos_coef + lcol[nonzero_index]'neg_coef
+    return T(sup)
+end
+
+function single_row_strengthening(row::RowOrCol{T}, b::T, m::Int, j::Int, ucol::Vector{T}, lcol::Vector{T}) where {T}
+    # perform coef strengthening for one constraints of the from a'x < b
+    # and return new a' and b'
+    flag = false # wether there is an update
```

Comment:
Two spaces after the end of the line + typo
```suggestion
    flag = false  # whether there is an update
```